templates/main/group_overview.html.ep

```diff
@@ -7,7 +7,17 @@
     $('.timeago').timeago();
 % end
 
-<h2>Last Builds for Group <%= $group->{name} %></h2>
+<h2>
+    Last Builds for Group
+    %= $group->{name}
+    % if (is_admin) {
+    <form action="<%= url_for('admin_job_templates', groupid => $group->{id}) %>" class="corner-buttons">
+        <button type="submit" class="btn btn-default" title="Edit medias, description, and other properties">
```

Comment:
"media" is already a plural word. These young guys with no proper humanistic education ;-) "one medium, multiple media" :-)